README.md

```diff
@@ -108,11 +108,63 @@ before starting kube-apiserver. For that you need to perform the following high
 Note: These steps have been verified with [kops](https://github.com/kubernetes/kops) but 
 it should be similar to any other cluster bootstrapping tool.
 
-#### Run aws-encryption-provider as static pod
-You need to have encryption provider running before kube-apiserver, and to do that you can
-use [static pods](https://kubernetes.io/docs/tasks/administer-cluster/static-pod/) functionality. For kops, static pod manifests are available at `/etc/kubernetes/manifests`. You can further use kops file assets functionality to drop 
-the pod spec file in that directory.
+#### Set the `--encryption-provider` flag with kops
```

Comment:
Can you migrate this guide for kops to a second file? I think the static pod directions are still valuable for non-kops deployment

Reply:
Can you please comment on the current structure. I left the guide on how to check if the provider is working correctly since that seems applicable to everyone.